_src/css/_body.scss

```diff
@@ -16,7 +16,29 @@
     margin: 25px;
     width: 50%;
     display: flex;
+  }
+
+  &__frame {
+    display: flex;
+    align-items: center;
+    padding-left:25 px;
```

Comment:
This padding rule doesn't work the way it's been declared, thus there's no need for it to be there